src/app/app-routing.module.ts

```diff
@@ -3,6 +3,7 @@ import { RouterModule, Routes } from '@angular/router';
 import { AppComponent } from './app.component';
 import { IndexComponent } from './components/index/index.component';
 import { ItinerarylistComponent } from './components/itinerary/itinerarylist/itinerarylist.component';
+import { ItineraryViewContoller } from './components/itinerary/itineraryshow/itineraryview.component';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/ispp-yourney/angular-app/pullRequest?prid=7181090)
```suggestion
import { ItineraryViewContoller } from "./components/itinerary/itineraryshow/itineraryview.component";
```